docs/source-control/new-repo.md

```diff
@@ -0,0 +1,46 @@
+---
+id: new-repo
+slug: /source-control/new-repo
+title: New Repository
+
+---
+
+## Basic Setup
+
+* Initialize the repository with `git init` (cli)
+* Create a new repository on github (github)
+* Select The Art of Education as the owner (github)
+* Select the proper repo privacy (github)
+
+![Github New Repo](github-new-repo.png)
+
+* Set the origin with `git remote add origin git@github.com:theartofeducation/your-repo-here.git` (cli)
+* create your initial commit and push (cli)
+
+## Common Linting Dependencies
+
+* All projects will require common linting dependencies and these are all written in JavaScript
```

Comment:
🤔 Are there any specific ways we should install these dependencies? For instance, when to do a "yarn install" vs. an "npm install"?

Reply:
you should always do a `yarn install`